esp_jpeg/test_apps/sdkconfig.defaults

```diff
@@ -2,3 +2,5 @@
 # Espressif IoT Development Framework (ESP-IDF) 5.4.0 Project Minimal Configuration
 #
 CONFIG_ESP_TASK_WDT_INIT=n
+CONFIG_JD_USE_ROM=n
```

Comment:
Should we also test with CONFIG_JD_USE_ROM=y?

Reply:
This repository is not setup for multiple configurations (sdkcongi.ci.something) as in esp-idf. I tried to add some minimal code to add this feature but it broke sh2lib examples.
So I think it is out of scope of this PR